packages/polka/src/index.ts

```diff
@@ -90,30 +95,34 @@ export default class Polka implements IModule, IServerHandler {
         // Handled errors are going to be instance of the ResponseError class
         if (err instanceof ResponseError) {
           // Handled errors are logged as warn as they are mostly harmless
-          logger?.warn(err);
-          return err.send(res);
+          logger?.warn({ err, traceId });
```

Comment:
We shouldn't need this. Instead can use logger library to handle this.

Reply:
What do you mean? I'm just using the same logger you were using, right?

Reply:
Sorry. What I meant was. The logger is `@cortec/logger` which is special implementation of `pino`. You can think about a more cleaner interface to log instead of changing the Error object to a plain object.